### ISSUE TITLE
Speech features not using dataset statistics

### ISSUE THREAD

**Issue by jangalt** [NVIDIA/OpenSeq2Seq#422]
When calculating the speech features for the speech2text models, OpenSeq2Seq calculates a mean and stddev individually for each training sample. Much like batch normalization, during inference, it would be better to calculate these constants against the training set to improve validation error. For example, consider the case of a speech input with 2 speakers, during training you will probably slice these into separate utterances, and the mean/stddev will track each utterance individually. If you perform inference against this however, the mean/stddev will get averaged, and not give the same results.

https://github.com/NVIDIA/OpenSeq2Seq/blob/bcdb76bb1221ebbef8eb2cd3cecaa491f801f54d/open_seq2seq/data/speech2text/speech_utils.py#L398-L400

**Comment by vsl9** [comment]
Thank you for the suggestion. Sure, exactly. We are aware of limitations of the current signal preprocessing pipeline. While it is fine for LibriSpeech and short audio clips in general, inference on longer utterances (and streaming ASR) requires different normalization schemes. We are working on that.

**Comment by gioannideImp** [comment]
So for long audio clips (for example call recordings where the form of speech is conversational and sometimes slang is used) would you suggest to break the audio file into smaller clips and transcribe each one individually? Or is there another suggestion you would recommend for long call recording audio files? Thanks.
